contracts/package.json

```diff
@@ -41,6 +41,6 @@
   },
   "engines": {
     "npm": ">6.0.0",
-    "node": ">=18.16.0"
+    "node": ">=18.15.0"
```

Comment:
I had to downgrade due to https://github.com/NomicFoundation/hardhat/issues/3877